src/server/mod.rs

```diff
@@ -42,14 +42,42 @@ impl SpaceapiServerBuilder {
 
         SpaceapiServerBuilder {
             status,
-            redis_connection_info: Err("redis_connection_info missing".into()),
+            pool: Err("redis_connection_info missing".into()),
             sensor_specs: vec![],
             status_modifiers: vec![],
         }
     }
 
     pub fn redis_connection_info<R: IntoConnectionInfo>(mut self, redis_connection_info: R) -> Self {
-        self.redis_connection_info = redis_connection_info.into_connection_info().map_err(|e| e.into());
+        let redis_connection_info  = redis_connection_info
+            .into_connection_info();
+
+        // Log some useful debug information
+        debug!("Redis connection info: {:?}", &redis_connection_info);
```

Comment:
Does this include the password? If yes, I don't think we should log it.

I'd suggest the following:

```suggestion
        debug!("Connecting to redis database {} at {}",
               redis_connection_info.db, redis_connection_info.addr);
```